content-warehouse/src/main/java/contentwarehouse/v1/QuickStart.java

```diff
@@ -0,0 +1,130 @@
+import com.google.cloud.contentwarehouse.v1.RequestMetadata;
+import com.google.cloud.contentwarehouse.v1.TextArray;
+import com.google.cloud.contentwarehouse.v1.TextTypeOptions;
+import com.google.cloud.contentwarehouse.v1.UserInfo;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+public class QuickStart {
+  
+  public static void main(String[] args)
+      throws IOException, InterruptedException, ExecutionException, TimeoutException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectNumber = "your-project-number";
+    String location = "your-region"; // Format is "us" or "eu".
+    String userId = "your-user-id"; // Format is user:<user-id>
+    quickStart(projectNumber, location, userId);
+  }
+
+  public static void quickStart(String projectId, String location, String userId)
+      throws IOException, InterruptedException, ExecutionException, TimeoutException {
+
+    String endpoint = String.format("%s-contentwarehouse.googleapis.com:443", location);
+    DocumentSchemaServiceSettings documentSchemaServiceSettings = 
+         DocumentSchemaServiceSettings.newBuilder().setEndpoint(endpoint).build(); 
+  
+    // Create a Schema Service client
+    try (DocumentSchemaServiceClient documentSchemaServiceClient =
+        DocumentSchemaServiceClient.create(documentSchemaServiceSettings)) {
+      /*  The full resource name of the location, e.g.:
+      projects/{project_number}/locations/{location} */
+      String parent = LocationName.format(projectId, location);
+
+      // Create Document Schema with Text Type Property
+      DocumentSchema documentSchema = DocumentSchema.newBuilder()
+          .setDisplayName("My Test Schema")
+          .setDescription("My Test Schema's Description")
+          .addPropertyDefinitions(
+            PropertyDefinition.newBuilder()
+            .setName("stock_symbol")
+            .setDisplayName("Searchable text")
+            .setIsSearchable(true)
+            .setTextTypeOptions(TextTypeOptions.newBuilder().build())
+            .build()).build();
+
+      // Define Document Schema request
+      CreateDocumentSchemaRequest createDocumentSchemaRequest =
+          CreateDocumentSchemaRequest.newBuilder()
+            .setParent(parent)
+            .setDocumentSchema(documentSchema).build();
+
+      // Create Document Schema
+      DocumentSchema documentSchemaResponse =
+          documentSchemaServiceClient.createDocumentSchema(createDocumentSchemaRequest); 
+        
+
+      // Create Document Service Client Settings
+      DocumentServiceSettings documentServiceSettings = 
+          DocumentServiceSettings.newBuilder().setEndpoint(endpoint).build();
+      TextArray textArray = TextArray.newBuilder().addValues("GOOG").build();
+
+      // Create Document Service Client and Document with relevant properties 
+      try (DocumentServiceClient documentServiceClient =
```

Comment:
It looks like this `try` for `DocumentServiceClient` is nested within the `try` for `DocumentSchemaServiceClient`. Should these two `try` blocks be nested? @averikitsch  what do you think?

Reply:
This pattern is fine.